src/hotspot/share/runtime/stubRoutines.cpp

```diff
@@ -394,7 +394,7 @@ void StubRoutines::initialize2() {
 
   // test safefetch routines
   // Not on Windows 32bit until 8074860 is fixed
-#if ! (defined(_WIN32) && defined(_M_IX86))
+#if ! (defined(_WIN32) && defined(_M_IX86)) && !defined(_M_ARM64)
```

Comment:
The comment needs updating to refer to Aarch64.

Reply:
This is actually not needed anymore, as it does work just fine on Windows+Aarch64. Presumably we have added it in the early days of the port when we haven't figured out exception handling quite yet.

Thanks for catching David.